### PR TITLE
Scrape Google Job Site

### LINKED ISSUE #71: [Backend] Scrape Google
## Description
We want to be able to successfully *and* consistently scrape job postings from Google's Careers site
## Tasks
- [ ] Scrape job postings From Google's Careers Site
- [ ] Parse data from Google job post

## To Discuss
Same as #68

### PR DESCRIPTION
**Pull request by a-camarillo**
#### What is the purpose of your change?
- fixes #71 
- Add script for scraping Google careers site, parsing job posts, and returning all of the necessary information for the job post

#### What areas of the application does this change impact? (changes API, folder structure, etc)
- Backend/Scraper

#### What features/fixes are deferred/not handled in this PR?
- Sending scraped data to the database and the associated API functionality for retrieving the data

## Checklist
- [ ] Adds unit tests
- [x] Has a work item linked to the PR

### CODE REVIEW

File: scraper/google.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+from selenium.webdriver.support.wait import WebDriverWait
+
+# Firefox is an alias for the WebDriver object from
+# selenium.webdriver.firefox.webdriver
+
+options = FirefoxOptions()
+options.add_argument("-headless")
+driver = Firefox(options)
+
+google_site = "https://careers.google.com/jobs/results/?distance=50&employment_type=INTERN&location=United%20States&location=Canada"
```
Comment (jwu910) [inline scraper/google.py:15]: How will distance apply when considering Us and Canada btw?

File: scraper/google.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        A reference link to a Google job posting
+
+    Returns
+    -------
+    job_info : dict{}
+        A dictionary containg all of the job information parsed from a
+        Google job posting
+    """
+
+    base = "https://careers.google.com"
```
Comment (jwu910) [inline scraper/google.py:183]: Can we push this base variable up to a constant or to a global variable? We use this domain url further up too I think

File: scraper/requirements.txt
Comment (jwu910) [inline scraper/requirements.txt:1]: Is this new? Did we forget to commit one before?

**Review by jwu910 — APPROVED** [review]
Couple questions, non blocking. Can we get some samples or demo screens as well?

File: scraper/requirements.txt
Comment (a-camarillo) [reply scraper/requirements.txt:1]: There was a blank .txt file committed.

File: scraper/google.py (hunk 2 above)
Comment (a-camarillo) [reply scraper/google.py:183]: sure thing

File: scraper/google.py (hunk 1 above)
Comment (a-camarillo) [reply scraper/google.py:15]: honestly not sure, `distance=50` is a default parameter applied by Google when you start adding filters to their job search site

**Comment by a-camarillo** [comment]
@jwu910 I can screen record a demo, where would we want it shared.

File: scraper/google.py (hunk 1 above)
Comment (jwu910) [reply scraper/google.py:15]: Maybe we want to max this out or unset it/remove it?

**Comment by jwu910** [comment]
> @jwu910 I can screen record a demo, where would we want it shared.
> 
> 
> 
> 

Here is fine, keeps context and proof of concept together, helps with the review

**Comment by a-camarillo** [comment]
@jwu910 here are some screenshots of the raw data and json formatted
image
image
image

**Pull request merged**